pkg/registry/core/namespace/storage/storage_test.go

```diff
@@ -74,7 +74,7 @@ func TestCreateSetsFields(t *testing.T) {
 	defer server.Terminate(t)
 	defer storage.store.DestroyFunc()
 	namespace := validNewNamespace()
-	ctx := genericapirequest.NewContext()
+	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), metav1.NamespaceNone)
```

Comment:
this is correct, because a namespace create request doesn't have `namespaces/$namespace` in the URL